plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -624,6 +626,7 @@ private static Path createSegment(InputStream tableConfigInputStream, InputStrea
         try {
             org.apache.pinot.spi.data.Schema pinotSchema = org.apache.pinot.spi.data.Schema.fromInputStream(pinotSchemaInputStream);
             TableConfig tableConfig = inputStreamToObject(tableConfigInputStream, TableConfig.class);
+            IngestionConfig ingestionConfig = tableConfig.getIngestionConfig();
```

Comment:
This changed caused NPE:
```
Error:  io.trino.plugin.pinot.TestPinotSecuredConnectorSmokeTest -- Time elapsed: 73.70 s <<< ERROR!
java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "org.apache.pinot.spi.config.table.ingestion.IngestionConfig.getBatchIngestionConfig()" because "ingestionConfig" is null
	at io.trino.plugin.pinot.BasePinotConnectorSmokeTest.createSegment(BasePinotConnectorSmokeTest.java:673)
	at io.trino.plugin.pinot.BasePinotConnectorSmokeTest.createAndPopulateMixedCaseHybridTablesAndTopic(BasePinotConnectorSmokeTest.java:465)
	at io.trino.plugin.pinot.BasePinotConnectorSmokeTest.createQueryRunner(BasePinotConnectorSmokeTest.java:151)
	at io.trino.testing.AbstractTestQueryFramework.init(AbstractTestQueryFramework.java:113)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
Caused by: java.lang.NullPointerException: Cannot invoke "org.apache.pinot.spi.config.table.ingestion.IngestionConfig.getBatchIngestionConfig()" because "ingestionConfig" is null
	at io.trino.plugin.pinot.BasePinotConnectorSmokeTest.createSegment(BasePinotConnectorSmokeTest.java:643)
	... 10 more
```